shap/__init__.py

```diff
@@ -33,24 +32,26 @@ def __getattribute__(self, item):
 
 
 try:
-    import matplotlib
+    import matplotlib  # noqa: F401
     have_matplotlib = True
 except ImportError:
     have_matplotlib = False
 if have_matplotlib:
     from . import plots
+    from .plots._bar import bar_legacy as bar_plot
     from .plots._beeswarm import summary_legacy as summary_plot
-    from .plots._decision import decision as decision_plot, multioutput_decision as multioutput_decision_plot
-    from .plots._scatter import dependence_legacy as dependence_plot
-    from .plots._force import force as force_plot, initjs, save_html, getjs
+    from .plots._decision import decision as decision_plot
+    from .plots._decision import multioutput_decision as multioutput_decision_plot
+    from .plots._embedding import embedding as embedding_plot
+    from .plots._force import force as force_plot
+    from .plots._force import getjs, initjs, save_html
+    from .plots._group_difference import group_difference as group_difference_plot
     from .plots._image import image as image_plot
     from .plots._monitoring import monitoring as monitoring_plot
-    from .plots._embedding import embedding as embedding_plot
     from .plots._partial_dependence import partial_dependence as partial_dependence_plot
-    from .plots._bar import bar_legacy as bar_plot
-    from .plots._waterfall import waterfall as waterfall_plot
-    from .plots._group_difference import group_difference as group_difference_plot
+    from .plots._scatter import dependence_legacy as dependence_plot
     from .plots._text import text as text_plot
+    from .plots._waterfall import waterfall as waterfall_plot
 else:
     summary_plot = unsupported
     decision_plot = unsupported
```

Comment:
could you also rearrange the variables within this else to match the new import order from above?